src/styles/global.css

```
@@ -64,6 +64,12 @@
         isolation: isolate;
         z-index: 1;
     }
+
+    @supports (corner-shape: squircle) {
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)